SYCL/InorderQueue/in_order_kernels.cpp

```diff
@@ -109,6 +109,7 @@ int main() {
       std::cerr << "Total mismatch =  " << err_cnt << std::endl;
       return 1;
     }
+    free(A, ctx);
```

Comment:
shouldn't we free this memory in case of the test failures. see line 110?

Reply:
Moving the free to line 107 (right after the loop) should do the trick.

Reply:
Thanks. Good catch.
Done.